### ISSUE TITLE
Message threads are not displayed

### ISSUE THREAD

**Issue by sphh** [friedPotat0/Spam-Scores#18]
**Describe the bug**
I always display my messages in threads:
![Screenshot from 2021-04-01 11-25-24](https://user-images.githubusercontent.com/4856462/113273517-409dd000-92cc-11eb-839c-ffb64cb316f5.png)
but after installing `Spam-Scores` I cannot display them in threads anymore if I have the `Spam scores` column displayed.

**Reproduction steps**
1. Display your messages in threads.
2. Install `Spam-Scores` according to the instructions.
3. Add the `Spam scores` column.
4. The threading disappeared.

**Expected behavior**
I would expect, that the messages are still displayed in treads. If the thread is collapsed, either do not display any spam score or display the highest (this could be configurable).

**Thunderbird (please complete the following information):**
 - OS: Linux Mint 20.1
 - Version: 78.7.1 (64-bit)

**Spam Score add-on (please complete the following information):**
 - Version: [e.g. 1.1.5] 1.1.6

**Comment by sphh** [comment]
Update:

Not displaying the `Spam score` column does not give me the thread display back.

I then deinstalled the add-on, restarted TB and repaired the folder. Unfortunately the ability to display the message threads is gone. 😢

How can I get my beloved message threads back??

**Comment by friedPotat0** [comment]
I'm really sorry to hear that the message threading is not working anymore. I've tried the provided reproduction steps, but the threading is working fine with Spam Scores installed.

![image](https://user-images.githubusercontent.com/5374007/113302780-b02db280-9300-11eb-9e08-c94c7fdebb50.png)

It seems to be more of an unfortunate coincidence, from what I can see at the moment. It also looks like the repair step has not managed to group the messages correctly in your folder. Perhaps you can fix the error by updating Thunderbird and then repairing the folder once again. The latest Thunderbird version is 78.9.0.

**Comment by sphh** [comment]
Very strange!

When I added the `Spam score` column, the colum was before the thread column by default. It also disabled the sorting by thread.

In the end I removed the `Spam Scores` add-on, started with all add-ons disabled, sorted by treads, restarted Thunderbird and reinstalled the `Spam Scores` add on. Now the column appeared behind the threads column and everything is working as it should.

There is a very embarrassing twist to this story: I have the [`No message pane sort`](https://github.com/speedball2001/nompsort-mx) installed, which disables all left clicks to the column bar without any modifier. I forgot about it. That might have been the reason, why I could not sort by threads after the first installation. But I would never have expected that the installation of the `Spam Scores` add-on disables this sorting. Might have been some sideeffect of an other add-on. Who knows?

Since everything is now working, I close this issue. Sorry for having bothered you.
